var/spack/repos/builtin/packages/pdt/package.py

```diff
@@ -50,6 +52,9 @@ def configure(self, spec, prefix):
         else:
             raise InstallError('Unknown/unsupported compiler family')
 
+        if '+pic' in spec:
+            options.append('-useropt=-fPIC')
```

Comment:
```suggestion
            options.append('-useropt=' + self.compiler.pic_flag)
```
Every compiler uses a different flag.

Reply:
Done! Thanks for the suggestion @adamjstewart.